docs/migration-guides/1.17-1.18.md

```diff
@@ -0,0 +1,11 @@
+# Migrate from 1.17 to 1.18
+
+## Istio Kyma patch
+
+In Kyma 1.18, we migrate the logic from Istio Kyma patch to a [TestDefinition](https://kyma-project.io/docs/#details-testing-kyma), which resides in the Istio chart. After the upgrade, it is no longer needed to keep it on the cluster, so it can be safely deleted.
```

Comment:
```suggestion
In Kyma 1.18, we migrate the logic from the Istio Kyma patch to a [TestDefinition](https://kyma-project.io/docs/#details-testing-kyma) which resides in the Istio chart. After the upgrade, it is no longer needed to keep the Istio Kyma patch on the cluster, so it can be safely deleted.
```